bash_tests/hw_bluetooth_speaker.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# the following will send test.wav file to bluetooth speaker.
+# you need to change the AMC address to match your actual bluetooth speaker MAC
+
+aplay -D bluealsa:HCI=hci0,DEV=11:2C:33:A4:1E:5B,PROFILE=a2dp speech/test.wav
```

Comment:
any way not to hard-code the bluetooth address here? is there a way to pull it from somewhere? does it need to be specified or will `aplay` just do the right thing when the bluetooth speaker is connected? (I think it did the last time I tried…)

(the irony is that I didn't realize `aplay` could be invoked in this way, so this is actually really useful for me…)

Reply:
I will check if it is possible; otherwise I will add comment on how to identify your own MAC Address.